src/code/z_scene_table.c

```diff
@@ -1025,11 +1027,11 @@ void* sDCLavaFloorTextures[] = {
     gDCLavaFloor5Tex, gDCLavaFloor6Tex, gDCLavaFloor7Tex, gDCLavaFloor8Tex,
 };
 
-// Scene Draw Config 20
+// Scene Draw Config 20 - Dodongo's Cavern
 void func_80099878(GlobalContext* globalCtx) {
     u32 gameplayFrames;
     s32 pad;
-    Gfx* displayListHead = Graph_Alloc(globalCtx->state.gfxCtx, 6 * sizeof(Gfx));
+    Gfx* dlistBuffer = Graph_Alloc(globalCtx->state.gfxCtx, 2 * sizeof(Gfx[3]));
```

Comment:
I think `displayListHead` (or `dListHead`) is better here since it's used as a pointer to the head and incremented over the function, whereas "buffer" makes it sound like it's a pointer to a buffer which isn't entirely true

Reply:
unlike in other places this holds two display lists, not just one, so I thought "buffer" made more sense

Reply:
It can be seen like that, but I'd argue the variable still holds a pointer to the head of either display lists so a name like `dListHead` makes more sense either way. The alternative would probably be `gfx` if you want something more generic, but it seems unnecessary to me.